nav2_motion_primitives/CMakeLists.txt

```diff
@@ -70,7 +70,11 @@ install(DIRECTORY include/
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
+  find_package(ament_cmake_gtest REQUIRED)
+
   ament_lint_auto_find_test_dependencies()
+  ament_add_gtest(test_motion_primitives test/test_motion_primitives.cpp)
```

Comment:
Our convention is to do an `add_subdirectory(test)` here and then make a CMakeLists.txt in the test directory that has the details of the tests ie. the `ament_add_gtest` line and the `ament_target_dependencies` line.

Reply:
Pushed a change.